src/transactions/feed/TransactionFeedItemImage.tsx

```diff
@@ -51,7 +57,16 @@ function TransactionFeedItemImage(props: Props) {
     )
   }
   if (transactionType === 'TokenTransferV3') {
-    return <ContactCircle recipient={props.recipient} size={AVATAR_SIZE} />
+    switch (props.transactionSubtype) {
```

Comment:
why are we using the "subType" instead of the __typename here? the jumpstart transactions have a different __typename, it's confusing that we implicitly assume that they have the __typename of 'TokenTransferV3' here?

Reply:
Well... Actually the only tx that has `__typename` as `JumpstartDeposit` is the deposit...
The reason for this is because the `reclaim` and the `received` don't need the extra fields, so it was enough to use the `TokenTransferV3` schema.
I guess we can identify those txs also as `JumpstartTransactions`, so the code would be clearer here, but I think I prefer to use the new type to indicate there are new fields in the schema.

Reply:
ah, i understood the types incorrectly. i thought that all jumpstart transactions would have the same __typename but i see that was not your intention [here](https://github.com/valora-inc/blockchain-api/pull/380/files#diff-f25aebc206e96407bd5d65c2ec780b48aa1f7f04efb1b9c7ddd41c617f87b51aL178)